pycaret/time_series.py

```diff
@@ -679,72 +683,72 @@ def ensemble_model(
 ) -> Any:
 
     """
-    This function ensembles a given estimator. The output of this function is
-    a score grid with CV scores by fold. Metrics evaluated during CV can be
-    accessed using the ``get_metrics`` function. Custom metrics can be added
-    or removed using ``add_metric`` and ``remove_metric`` function.
+     This function ensembles a given estimator. The output of this function is
```

Comment:
Is the extra spacing in the docstring needed?

Reply:
Not sure what caused the extra spacing, I'm fixing this in the next commit. Thanks